<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graficador</class>
 <widget class="QWidget" name="Graficador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graficador</string>
  </property>
  <widget class="QGraphicsView" name="GrafoView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>851</width>
     <height>691</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>550</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditVerticeValor">
      <property name="placeholderText">
       <string>Valor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAgregarVertice">
      <property name="text">
       <string>Agregar Vertice</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPeso">
      <property name="placeholderText">
       <string>Peso</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditOrigen">
      <property name="placeholderText">
       <string>Origen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDestino">
      <property name="placeholderText">
       <string>Destino</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAgregarArista">
      <property name="text">
       <string>Agregar Arista</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Grafo</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="MatrizView">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>60</y>
     <width>421</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>550</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEditEliminarVertice">
      <property name="placeholderText">
       <string>Vertice</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEliminarVertice">
      <property name="text">
       <string>Borrar Vertice</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditEliminarAristaOrigen">
      <property name="placeholderText">
       <string>Origen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditEliminarAristaDestino">
      <property name="placeholderText">
       <string>Destino</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEliminarArista">
      <property name="text">
       <string>Borrar Arista</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnMatrizAdyacencia">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>430</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Calcular Matriz Adyacencia</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbAlgoritmos">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ninguno</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Floyd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Warshall</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dijkstra</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prim</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kruskal</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
